README.md

```diff
@@ -1,2 +1,36 @@
 # rca-barefoot-team6-bn
+
+# Barefoot
+
+Barefoot is a project which helps people book accomodation in places that they plan to stay at. 
+The application is mostly for tourists and generally anyone who wants to know and book accomodation facilities in an area when they plan to visit. 
+
+
+
+
+
+## Installation
+
+- Clone the project 
+
+- Install dependencies with
+```bash 
+npm install
+```
+- Create a ```bash .env``` file to the root directory of the project 
+- Copy content from ```bash .env.example``` to the ``` .env``` file
+- To each key, assign its corresponding value which will be used while running the project
+
+- Start the server using
+```bash
+npm run dev
+```
+
+## License
+
+[MIT](https://choosealicense.com/licenses/mit/)
+
+
+
 [![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
```

Comment:
Always badges came on the first section (May be from line 2); and also remember to add on Github actions badges

ex:
![image](https://user-images.githubusercontent.com/41394162/234521914-a6cf8c07-46b8-4d3b-9782-f22b974ea6e7.png)

Reply:
Also, these contents aren't enough; you need to add more such as contributors, TTL, manager, and Project owner as well

- Even you need to specify the technologies used.